### ISSUE TITLE
Landing page https://www.cessda.eu/Training

### ISSUE THREAD

**Issue by hirschlisa** [cessda/cessda.training-resources.issues#4]
Working group is still mentioned here, this should be deleted. 

link to the core working groups under governance (https://www.cessda.eu/Core-Working-Groups) instead as TR are now a service?

**Comment by alen-vodopijevec-cessda** [comment]
Would you rather mention something about the Service at bottom instead "Working group" block?

**Comment by hirschlisa** [comment]
Exactly, I'd suggest to add the link to the core working groups under governance (https://www.cessda.eu/Core-Working-Groups) where the training resources are listed instead as the Training Resources are now a service. I think ultimately users should not get confused about the different groups and being clear about who is currently responsible for what facilitates that understanding.

If the Training Working Group site as a whole should be deleted is not anything I feel I can decide, as this is pre service ownership on my part and a more general question regarding the structure.

**Comment by alen-vodopijevec-cessda** [comment]
Remove WG info and add "Contact the Service owner"

https://www.cessda.eu/Training

**Comment by hirschlisa** [comment]
@alen-vodopijevec-cessda are you editing the landing page - I do not see how I can change that and did not get any intro into editing the website.

**Comment by hirschlisa** [comment]
@alen-vodopijevec-cessda I see that the working group site itself is not reachable any more but the link to it is still on the landing page, could you look into that?

**Comment by hirschlisa** [comment]
@alen-vodopijevec-cessda just following up on this

**Comment by alen-vodopijevec-cessda** [comment]
> Remove WG info and add "Contact the Service owner"
> 
> https://www.cessda.eu/Training

@openconcept .. Can you remove the "Working group" block at the end of the page and change "Contact" text to "You can reach the Training Resources Service Owner by email at training@cessda.eu"?

**Comment by openconcept** [comment]
It's been updated!

**Comment by hirschlisa** [comment]
great, thanks!
